src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done